### PR TITLE
feature(ui-ux): ocg cfp/dfip proposals

### PR DESCRIPTION
**Pull request by JoshuaC817**
#### What this PR does / why we need it:
OCG feature for submitting CFP/DFIP proposals, for now mobile LW only supports submitting proposal, to view list of submitted proposals and vote proposals can only be done on scan.
example of a valid github url: https://github.com/defich/dfips/issues/122

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode\*
- [x] Your UI implementation visually matched the rendered design\*
- [x] Unit tests\*
- [x] Added e2e tests\*
- [x] Added translations\*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ProposalURLInput.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    trigger,
+    formState: { isValid },
+  } = useForm({
+    mode: "onChange",
+  });
+  const url = getValues("url");
+  const hasInput = url !== "" && url !== undefined;
+
+  useEffect(() => {
+    async function checkUrlValidity() {
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ProposalURLInput.tsx:36]: do we have any promises in this function?

Comment (JoshuaC817) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ProposalURLInput.tsx:36]: nope

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/CfpDfipProposalsFaq.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    {
+      title: translate(
+        "components/CfpDfipProposalsFaq",
+        "Can I vote on proposals using Light Wallet?"
+      ),
+      content: [
+        {
+          text: translate(
+            "components/CfpDfipProposalsFaq",
+            "No, only proposal submissions can be done through Light Wallet. Currently, voting can only be submitted on chain using commands via CLI. Voting commands can be generated on DeFiscan via On-Chain Governance tab > Proposal details > Submit vote.\nYou will need to own a full node wallet and at least one masternode to vote."
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/CfpDfipProposalsFaq.tsx:65]: let's use this casing, and let's update all translation `.json` too
```suggestion
            "No, only proposal submissions can be done through Light Wallet. Currently, voting can only be submitted on chain using commands via CLI. Voting commands can be generated on DeFiScan via On-Chain Governance tab > Proposal details > Submit vote.\nYou will need to own a full node wallet and at least one masternode to vote."
```

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ProposalURLInput.tsx (hunk 1 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ProposalURLInput.tsx:36]: then do we need `async`?

Comment (JoshuaC817) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/components/ProposalURLInput.tsx]: ah good catch, removed `async`, thanks!

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/CfpDfipProposalsFaq.tsx (hunk 2 above)
Comment (JoshuaC817) [reply mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/CfpDfipProposalsFaq.tsx]: Updated to the correct casing

**Pull request merged**